rfc6962-bis.xml

```diff
@@ -784,6 +784,12 @@ messages.
         In general, where needed, the <spanx style="verb">version</spanx> is v1 and the <spanx style="verb">id</spanx> is the log id for the log server queried.
       </t>
       <t>
+	In practice, log servers may be comprised of multiple machines. Since it is impractical to keep these machines in perfect sync, errors may occur that are caused by skew between the machines. Where such errors are possible, the log server will return additional information making it possible for clients to make progress, if progress is possible. Servers MUST only serve data that is free of gaps (that is, for example, no server will respond with an STH unless it is also able to prove consistency from all log entries logged within that STH).
```

Comment:
re ... the log servers will return additional information ...:
Is this optional? Can you also clarify how it should be done?